README.md

```diff
@@ -15,47 +15,43 @@ Add the plugin and repositories listing to `project.clj`. **NB: You need to add
 :plugins [[s3-wagon-private "1.2.0"]]
 ```
 
-You can store credentials either in an encrypted file or as
-environment variables. For the encrypted file, add this to
-`project.clj`:
+To authenticate to the S3 bucket, you can either use any of the AWS SDK credential providers or store credentials in an encrypted file.
 
-```clj
-:repositories [["private" {:url "s3p://mybucket/releases/" :creds :gpg}]]
-```
+- Using one of the AWS SDK [chained provider class][chained-provider-class] credential providers:
 
-And in `~/.lein/credentials.clj.gpg`:
+     Add the following to `project.clj`:
 
-```
-{"s3p://mybucket/releases" {:username "AKIA2489AE28488"
-                            :passphrase "98b0b104ca1211e19a6c"}}
-```
+     ```clj
+     :repositories [["private" {:url "s3p://mybucket/releases/" :no-auth true}]]
+     ```
 
-The username and passphrase here correspond to the AWS Access Key and Secret
-Key, respectively.
+     An excerpt of the most commonly used credential providers:
+     - Environment Variables - `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`
+     - Java System Properties - `aws.accessKeyId` and `aws.secretKey`
+     - Credential profiles file at the default location (~/.aws/credentials) with the [AWS Credentials File Format][credentials-file-format]. To use a particular profile, specify the env var `AWS_PROFILE` or a Java system property of `aws.profile`, otherwise, the fallback will be the default profile name (`"default"`)
```

Comment:
@rymndhng Just to sanity check, this is with s3-wagon-private 1.3.0-alpha1? If so, can you provide an example or more details of what you're seeing? I just retried and was able to successfully use `AWS_PROFILE` to switch which access key I wanted to use. 

As an example, I created a credential profiles file in `~/.aws/credentials` and put a default profile with a valid key and a "profile2" with an invalid key. Running `AWS_PROFILE=profile2 lein deps` gives the expected error of `The AWS Access Key Id you provided does not exist in our records.` Specifying the default profile, however, with `AWS_PROFILE=default lein deps` retrieves the dependency as expected.